docs/contracts.rst

```diff
@@ -584,6 +587,87 @@ Please ensure you test your fallback function thoroughly to ensure the execution
         }
     }
 
+.. _overload-function:
+
+Function Overloading
+====================
+A Contract can have multiple functions of the same name but with different arguments.
+This also applies to inherited functions. The following example shows overloading of the
+``Z`` function in the scope of contract ``A``.
+
+::
+
+    pragma solidity ^0.4.0;
+
+    contract A {
+      function Z(uint _in) public pure returns (uint out)
```

Comment:
Please follow the indentation from the other examples (well, from the style guide if possible) and use a lower case function name.

Convention was so far to use `contract C { function f(); function g(); ... }`.